README.md

```diff
@@ -1,4 +1,6 @@
 [![Build Status](https://travis-ci.org/hmcts/draft-store.svg?branch=master)](https://travis-ci.org/hmcts/draft-store)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/35eb37f39906421387cfd120c35a538d)](https://www.codacy.com/app/HMCTS/draft-store?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=hmcts/draft-store&amp;utm_campaign=Badge_Grade)
```

Comment:
I am not convinced about tracking thing. Do we really want to do this?

Reply:
I would remove the GA link